### DISCUSSION TITLE
Added a Video to how to setup the project on README :partying_face:  !!!

### DISCUSSION THREAD

**Discussion by jvJUCA** [ruxailab/RUXAILAB#325] [Announcements]
<a href='https://youtu.be/dAf4LRxITCc'>Tutorial - Running RUXAILAB with Firebase Emulators   🖥️</a>

**Comment by pankaj87072** [comment]
hey @jvJUCA can't we contribute in 'web-eye-tracker-front' repo??

**Reply by jvJUCA** [reply]
Dear @pankaj87072 ,

Your willingness to contribute is highly appreciated. You are more than welcome to contribute to any repository within our organization. Your input and efforts are valuable to us, and we encourage you to engage with any projects you find interesting or feel you can assist with.

Thank you for being open to collaboration and helping us make progress together. 

Best regards,

@jvJUCA

**Reply by pankaj87072** [reply]
@jvJUCA thanks for clarification

**Comment by KarinePistili** [comment]
I'll be closing this discussion as the video was already shared on readme.
